src/test/java/nl/tudelft/contextproject/util/SizeTest.java

```diff
@@ -50,7 +49,7 @@ public void testEqualsEqual() {
 	 */
 	@Test
 	public void testEqualsSame() {
-		assertTrue(size.equals(size));
+		assertEquals(size, size);

```

Comment:
Even though this test does the exact same thing the assertTrue made it clearer that we were testing our equals method. This just raises the question if this is even a valid test, as comparing if something is equal to itself should be true.